src/screens/UserPortal/People/People.spec.tsx

```
@@ -519,9 +653,9 @@ describe('Testing People Screen Pagination [User Portal]', () => {
   });
 
   it('handles backward pagination correctly', async () => {
-    // Use mocks that support forward and backward navigation
+    // Initial load + useEffect refetch both use same vars; then fetchMore uses after: 'cursor3'
     render(
-      <MockedProvider mocks={[defaultQueryMock, nextPageMock]}>
+      <MockedProvider mocks={[defaultQueryMock, defaultQueryMock, nextPageMock]}>
```

ESLint

[error] Replace `·mocks={[defaultQueryMock,·defaultQueryMock,·nextPageMock]}` with `⏎········mocks={[defaultQueryMock,·defaultQueryMock,·nextPageMock]}⏎······`

(prettier/prettier)